.circleci/generate.sh

```diff
@@ -34,6 +34,7 @@ print_help()
   echo "   -a Generate the config.yml, config.yml.FREE and config.yml.PAID expanded configuration"
   echo "      files from the main config_template.yml reusable configuration file."
   echo "      Use this for permanent changes in config.yml that will be committed to the main repo."
+  echo "   -d Minimal development checks only. Sanity check during your dev before sending it to review for speed and cost reductions."
```

Comment:
nit that can be addressed on commit: `We should add: Submitting cleaning pre-commit clean CI run is still a requirement when the patch is ready for review.`
Same for trunk

Reply:
Added to PR